docs/sources/administration/provisioning/index.md

```diff
@@ -190,6 +190,8 @@ Since not all datasources have the same configuration settings we only have the
 | maxIdleConns               | number  | MySQL, PostgreSQL and MSSQL                                      | Maximum number of connections in the idle connection pool (Grafana v5.4+)                                                                                                                                                                                                                                           |
 | connMaxLifetime            | number  | MySQL, PostgreSQL and MSSQL                                      | Maximum amount of time in seconds a connection may be reused (Grafana v5.4+)                                                                                                                                                                                                                                        |
 | keepCookies                | array   | _HTTP\*_                                                         | Cookies that needs to be passed along while communicating with datasources                                                                                                                                                                                                                                          |
+| prometheusVersion          | string  | Prometheus                                                       | The version of the Prometheus datasource (e.g. `2.37.0`, `2.24.0`)                                                                                                                                                                                                                                                  |
+| prometheusType             | string  | Prometheus                                                       | The type of the Prometheus datasources (i.e. `Prometheus`, `Cortex`, `Thanos`, or `Mimir`)                                                                                                                                                                                                                          |
```

Comment:
We should also note these settings, as well as the provisioning options and when/why they're required, in [the data source's own documentation](https://github.com/grafana/grafana/blob/main/docs/sources/datasources/prometheus.md).

Reply:
@gguillotte-grafana Good catch! I have implemented a blurb on that page as well for the new values. Can you take a look and let me know what you think? https://github.com/grafana/grafana/pull/56496/commits/e90d133a3a92bf2851176a5c8bb88e6d2bae638d
Sorry the diff is bad, I increased the max line width and that wound up adding spaces to every line in that block

Reply:
LGTM